contracts/starknet/lib/general_address.cairo

```diff
@@ -0,0 +1,4 @@
+# Generalized type used to represent addresses in Snapshot X. Eg Ethereum, Starknet, etc.
+struct Address:
+    member value : felt
+end
```

Comment:
Should this be called `lib/address` ? I find it odd to have a filename be `general_address` and the actual object be `Address` ?

Reply:
hmm yeah agree, just thought that people would assume its a Starknet address if its called `address.cairo`